api/files.js

```diff
@@ -50,3 +54,88 @@ module.exports.writeMail = function (recipients, template, substitutions) {
 	mkdirp.sync(MAIL_DIRECTORY);
 	return fs.writeFileAsync(filePath, fileContent);
 };
+
+/**
+ * Writes a file to the temp directory. When called in non-development environments,
+ * this method does nothing
+ * @param {Buffer} content		the content to write
+ * @param {Object} params		a parameter object with
+ *                         		{String} key	the key under which to store the file
+ *                         		{String} bucket	the bucket in which to store the file
+ *                         		{String} type	the MIME type of the file
+ * @return {Promise<>}			a resolved promise
+ */
+module.exports.writeFile = function (content, params) {
```

Comment:
Just a thought but we might want a better indication that it only works for the "Development" mode by adding a prefix to the function. So instead of `writeFile()` we might want to call it `devModeWriteFile()`